client/bin/entrypoint.sh

```diff
@@ -31,4 +31,8 @@ sed --follow-symlinks -i "s/KOJI_HUB_IP/${WORKSTATION_IP}/" /etc/osbs.conf
 sed --follow-symlinks -i "s/OPENSHIFT_IP/${WORKSTATION_IP}/" /etc/osbs.conf
 sed --follow-symlinks -i "s/REGISTRY_IP/${WORKSTATION_IP}/" /etc/osbs.conf
 
+# Replace koji IP
+KOJI_IP=$(getent hosts koji-hub | awk '{ print $1 }')
```

Comment:
A couple of lines above, we set the `KOJI_HUB_IP` in osbs.conf to be `WORKSTATION_IP`.
Why not use that?